File: Beakjoon/_etc/Java/2875 대회 or 인턴.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      int n = sc.nextInt(); int m = sc.nextInt(); int k = sc.nextInt();
      int team = 0;
      while(true) {
          if(n < 2 || m < 1) break;
          n -= 2; m-= 1; team += 1;
      }
      int res = n + m;
      while(true) {
          if(res >= k) break;
          res += 3; team -= 1;
      }
      System.out.println(team);
    }
}

File: Beakjoon/_etc/Kotlin/2875 대회 or 인턴.kt
import java.util.*
import kotlin.math.*

fun main(args: Array<String>) {
    var sc = Scanner(System.`in`)
    var n = sc.nextInt(); var m = sc.nextInt(); var k = sc.nextInt(); var team = 0;
    while(true) {
        if(n < 2 || m < 1) break
        n -= 2; m--; team++;
    }
    var res = n+m
    while(true) {
        if(res >= k) break
        res += 3; team--
    }
    println(team)    
}
